commercetools-internal-processors/src/test/java/io/sphere/sdk/annotations/processors/generators/UpdateActionGeneratorTest.java

```diff
@@ -0,0 +1,84 @@
+package io.sphere.sdk.annotations.processors.generators;
+
+import com.google.common.base.Charsets;
+import com.google.testing.compile.CompilationRule;
+import com.squareup.javapoet.JavaFile;
+import io.sphere.sdk.annotations.processors.generators.examples.ExampleResourceWithUpdateAction;
+import org.apache.commons.io.IOUtils;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+
+import javax.lang.model.element.ExecutableElement;
+import javax.lang.model.element.TypeElement;
+import javax.lang.model.util.ElementFilter;
+import java.io.InputStream;
+import java.util.Optional;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+
+public class UpdateActionGeneratorTest  {
+
+    @Rule
+    public CompilationRule compilationRule = new CompilationRule();
+
+    protected AbstractGenerator generator;
+
+    @Before
+    public void setup() {
+        generator = new UpdateActionGenerator(compilationRule.getElements(), compilationRule.getTypes(),null);
+    }
+
+    @Test
+    public void generateExampleUpdaupdateActionNameteActionForNonNullable() throws Exception {
```

Comment:
I guess that this method should be called `generateExampleUpdateActionNameActionForNonNullable`? 😉